### ISSUE TITLE
Output files too large.

### ISSUE THREAD

**Issue by JdrrCosta** [fgvieira/ngsLD#54]
I first ran the software with the geno files produced by ANGSD with -doGeno 32

```
cat all_chr_sites | xargs -n 2 -P 2 sh -c '/home/jcosta/software/ngsLD/ngsLD --geno Lcas_chr_$0_input_ngsLD.geno --out Lcas_chr_$0_output_ngsLDld --pos Lcas_chr_$0_input_ngsLD.pos \
--n_ind 22 --n_sites $1 --verbose 1 --probs --max_kb_dist 1000 --min_maf 0.068 --rnd_sample 0.05 --seed 1'
```
Then since I wanted to use only the genotype likelihoods and not called genotypes I ran a second time withe the glf files produced by ANGSD with the flag -doGlf 3.

```
cat all_chr_sites | xargs -n 2 -P 2 sh -c '/home/jcosta/software/ngsLD/ngsLD \
--geno Lcas_chr_$0_input_ngsLD.glf --out Lcas_chr_$0_GL1_output_ngsLDld.ld --pos Lcas_chr_$0_input_ngsLD.pos \
--n_ind 22 --n_sites $1 --verbose 1 --probs --log_scale --max_kb_dist 1000 --min_maf 0.068 --rnd_sample 0.05 --seed 1'
```
But this time each of the .ld files is over 300GB when the .ld files produced with the first command were less than 100Gb each.  Is this to be expected?

**Comment by fgvieira** [comment]
Can you post the first (e.g) 10 lines of both outputs?

**Comment by JdrrCosta** [comment]
Unfortunately all I have is this they are for two different species but i had to delete the outputs since they were taking too much space and the machine was getting full. The top one was run using the .geno file and the bottom one using the .glf. I can try to run again if this does not suffice. 
![image](https://github.com/fgvieira/ngsLD/assets/169660039/05bd643a-0148-4d35-9de7-335a65114ef1)

**Comment by fgvieira** [comment]
Can you try running a `geno` and a `glf` file of the same dataset and run them through `ngsLD`? So we can compare outputs, namely:
- number of lines
- number of columns
- float precision

**Comment by JdrrCosta** [comment]
Hello Again,
I am sorry for the delay responding but I lost access to my server temporarily. While preparing the runs for ngsLD to compare outputs as you suggested i noticed that the .glf file had many more sites than the .geno. Looking into it I realized I did not instate a minMaf filter in the glf file generation as I did in the .geno file. Instating the filter seems to solve the problem and give similar sized files. 
It was an oversight i should have noticed. Thank you for your help and time.
